.github/workflows/smoke-checks.yml

```diff
@@ -1,19 +1,10 @@
 name: Smoke Checks
 
 on:
-  push:
-    branches:
-      - main
-      - develop
-
   pull_request:
     branches:
       - '**'
 
-  release:
-    types:
-      - created
-
```

Comment:
Here as well?

Reply:
Why do we need to run some checks again after we executed them on release PR, then on push to `main`, and now on "release created" 🤔

Reply:
What do you mean by "after we executed them on release PR"?

Reply:
These checks should be run on the Release PR, that is my only concern that we are sure they are still run on Release PR

Reply:
All clarified now 👍